### PR TITLE
add tiflash faq

### PR DESCRIPTION
**Pull request by windtalker**
### What is changed, added or deleted? (Required)
add TiFlash faq

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by TomShawn** [comment]
@windtalker Does this PR also apply to v4.0?

**Comment by windtalker** [comment]
> @windtalker Does this PR also apply to v4.0 and v3.1?

Yes, but I thought this pr is for the docs special week, it can not be cherry picked to v4.0 and v3.1 directly?

**Comment by yikeke** [comment]
> > @windtalker Does this PR also apply to v4.0 and v3.1?
> 
> Yes, but I thought this pr is for the docs special week, it can not be cherry picked to v4.0 and v3.1 directly?

We will merge this `docs-special-week` branch into `master` and `release-4.0` branches later. If your PR changes also apply to other versions, you need to specify them in the PR description.

**Comment by windtalker** [comment]
> > > @windtalker Does this PR also apply to v4.0 and v3.1?
> > 
> > 
> > Yes, but I thought this pr is for the docs special week, it can not be cherry picked to v4.0 and v3.1 directly?
> 
> We will merge this `docs-special-week` branch into `master` and `release-4.0` branches later. If your PR changes also apply to other versions, you need to specify them in the PR description.

Actually, this pr is for the `docs-special-week`, so it only need to apply to the versions that `docs-special-week` branch affects.

**Comment by TomShawn** [comment]
LGTM

**Comment by yikeke** [comment]
> > > > @windtalker Does this PR also apply to v4.0 and v3.1?
> > > 
> > > 
> > > Yes, but I thought this pr is for the docs special week, it can not be cherry picked to v4.0 and v3.1 directly?
> > 
> > 
> > We will merge this `docs-special-week` branch into `master` and `release-4.0` branches later. If your PR changes also apply to other versions, you need to specify them in the PR description.
> 
> Actually, this pr is for the `docs-special-week`, so it only need to apply to the versions that `docs-special-week` branch affects.

TiFlash docs generally apply to master, v4.0, and v3.1 versions. I'll tick the three versions in the PR description box for you.

File: faq/tidb-faq.md [hunk 1]
```diff
@@ -695,28 +695,50 @@ TiKV 的内存占用主要来自于 RocksDB 的 block-cache，默认为系统总
 
 不可以。TiDB 数据（或使用其他事务 API 生成的数据）依赖于一种特殊的键值格式，和 RawKV API 数据（或其他基于 RawKV 的服务生成的数据）并不兼容。
 
-### 3.5 TiDB 测试
+### 3.5 TiFlash 管理
```
Comment (zanmato1984) [inline faq/tidb-faq.md:698]: I think just "TiFlash" is good for the title. "管理" doesn't quite apply here.

**Comment by windtalker** [comment]
> > > > > @windtalker Does this PR also apply to v4.0 and v3.1?
> > > > 
> > > > 
> > > > Yes, but I thought this pr is for the docs special week, it can not be cherry picked to v4.0 and v3.1 directly?
> > > 
> > > 
> > > We will merge this `docs-special-week` branch into `master` and `release-4.0` branches later. If your PR changes also apply to other versions, you need to specify them in the PR description.
> > 
> > 
> > Actually, this pr is for the `docs-special-week`, so it only need to apply to the versions that `docs-special-week` branch affects.
> 
> TiFlash docs generally apply to master, v4.0, and v3.1 versions. I'll tick the three versions in the PR description box for you.

But this is only a refine pr aimed at `docs-special-week`, v3.1 has its own `tiflash-faq` part, in my understanding, it should not be merged to v3.1

**Comment by zanmato1984** [comment]
@yikeke I think we'd better leave tiflash in 3.1 as is. So could you please uncheck the 3.1 branch?

**Comment by yikeke** [comment]
> @yikeke I think we'd better leave tiflash in 3.1 as is. So could you please uncheck the 3.1 branch?

done

**Pull request merged**